Components/WebService/Rest/RestConfigurableProtocol.php

```diff
@@ -15,6 +15,8 @@ class RestConfigurableProtocol extends ConfigurableWebserviceProtocol implements
     const OPTION_HEADERS = 'headers';
     const OPTION_AUTH = 'authentication';
     const OPTION_ENCODING = 'encoding';
+    /** @deprecated: Will be removed on next major version. Fallback will not happen in v3. */
```

Comment:
```suggestion
    /** @deprecated: It will be removed on next major version. Fallback will not happen in v3. */
```

Reply:
It's a null subject phrase in imperative form. There's no need for a subject: https://en.wikipedia.org/wiki/Null-subject_language#The_imperative_form

Reply:
Ok, when reading the first time I took as informative sentence rather than imperative